scripts/install/calibrecs.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+if [[ ! -f /install/.calibre.lock ]]; then
+    echo_error "Calibre Content server requires calibre. Please run \`box install calibre\` first."
+    exit 1
+fi
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+
+if [ -z "$CALIBRE_LIBRARY_USER" ]; then
+    if ! CALIBRE_LIBRARY_USER="$(swizdb get calibre/library_user)"; then
+        CALIBRE_LIBRARY_USER=$(_get_master_username)
+        swizdb set "calibre/library_user" "$CALIBRE_LIBRARY_USER"
+    fi
+fi
+
+if [ -z "$CALIBRE_LIBRARY_PATH" ]; then
+    if ! CALIBRE_LIBRARY_PATH="$(swizdb get calibre/library_path)"; then
+        CALIBRE_LIBRARY_PATH="/home/$CALIBRE_LIBRARY_USER/Calibre Library"
+        swizdb set "calibre/library_path" "$CALIBRE_LIBRARY_PATH"
+    fi
+fi
+
+clbServerPath="/home/$CALIBRE_LIBRARY_USER/.config/calibrecs"
+
+_adduser() {
+    echo_progress_start "Adding users to calibre content server"
+    for user in "${users[@]}"; do
+        echo_log_only "Adding user $user"
+        pass=$(_get_user_password "$user")
+        echo -e "1\n$user\n$pass\n$pass" | calibre-server --userdb "$clbServerPath"/server-users.sqlite --manage-users || {
+            echo_error "Something failed?"
+            exit 1
+        }
+    done
+    echo_progress_done "Users added to calibre content server"
+
+    chown -R "$CALIBRE_LIBRARY_USER": "$clbServerPath"
+}
+
+_systemd() {
+    cat > /etc/systemd/system/calibrecs.service << CALICS
+[Unit]
+Description=calibre content server
+After=network.target
+
+[Service]
+Type=simple
+User=$CALIBRE_LIBRARY_USER
+Group=$CALIBRE_LIBRARY_USER
+ExecStart=/usr/bin/calibre-server --max-opds-items=30 --max-opds-ungrouped-items=100 --port 8089 "${CALIBRE_LIBRARY_PATH:=CALIBRE_LIBRARY_PATH_GOES_HERE}"
```

Comment:
```
"${CALIBRE_LIBRARY_PATH:=CALIBRE_LIBRARY_PATH_GOES_HERE}"
```
🤨

Reply:
That's a default value that I tell the user in echos as well as docs to change manually in case the library variable could not be resolved